tests/KernelFlexibleCenterer.py

```diff
@@ -0,0 +1,71 @@
+import unittest
```

Comment:
file name should be `kernel_flexible_centerer.py`

Reply:
Done, rename everything